src/scripts/main.js

```diff
@@ -2,7 +2,48 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  const newTable = document.createElement('tr');
+
+  table.append(newTable);
+
+  const nameColumn = document.createElement('th');
+
+  nameColumn.textContent = person.name;
+  newTable.append(nameColumn);
+
+  const genderColumn = document.createElement('th');
+
+  switch (person.sex) {
+    case 'f':
+      person.sex = 'Female';
```

Comment:
You mutate properties of person doing it this way.

```suggestion
people.forEach(person => {
  let { sex } = person;
  const newTable = document.createElement('tr');

  table.append(newTable);

  const nameColumn = document.createElement('th');

  nameColumn.textContent = person.name;
  newTable.append(nameColumn);

  const genderColumn = document.createElement('th');

  switch (sex) {
    case 'f':
      sex = 'Female';
```